docs/compass/04-01-compass.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Enable Compass in Kyma
+type: Installation
+---
+
+To enable Compass in Kyma, follow the instructions for the [custom component installation](/root/kyma#configuration-custom-component-installation) and enable the `compass` and `compass-runtime-agent` modules.
+You can also [install Kyma on a cluster](/root/kyma#installation-install-kyma-on-a-cluster) with the ready-to-use configurations for different Compass modes.
+
+## Default Kyma installation
+
+This is a preconfigured single-tenant and single-Runtime mode which will eventually become part of the default Kyma installation. It provides the complete cluster Kyma installation with all components, including both Compass and Agent. To enable this mode, follow the cluster Kyma installation and use the [`installer-cr-cluster-with-compass.yaml.tpl`](https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-cr-cluster-with-compass.yaml.tpl) configuration file.
+
+## Kyma with Compass only
+
+This is a multi-tenant and multi-runtimes mode that provides cluster Kyma installation with Compass only. This configuration includes only the selected Kyma components that Compass uses. To enable this mode, create this ConfigMap and then perform the cluster Kyma installation using the
```

Comment:
I think it is not clear what `cluster Kyma installation with Compass only` means. It mentions `only` Compass but later it says that it installs `the selected Kyma components that Compass uses`.  I would also definitely rephrase it not to use "only" at the end of the sentence but will not propose anything as I would not like to change the meaning not knowing the original intention :)

Reply:
Compass only, meaning that it uses COmpass without Agent + the Kyma components are limited. Let's discuss it tomorrow :)